src/extras/primitives/primitives/a-text.js

```diff
@@ -0,0 +1,31 @@
+/* Experimental text primitive.
+ * Issues: color not changing, removeAttribute() not working, mixing primitive with regular entities fails
+ * Color issue relates to: https://github.com/donmccurdy/aframe-extras/blob/master/src/primitives/a-ocean.js#L44
```

Comment:
The problems with the text primitive do seem to have been fixed with A-Frame v0.4.0.

@donmccurdy with 0.3.0 adding a text primitive to a scene would sometimes break text added the regular way (e.g. `<a-entity bmfont-text="text: Hello World;"></a-entity>` would no longer show up)